plugins/utils/scope.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, helpers
```

Comment:
We've soft-deprecated using `helpers`. Instead you should use `lib.nixvim`, which is the same thing.